### PR TITLE
added header and nav to contact page

### PR DESCRIPTION
**Pull request by TobyLtt**

### CODE REVIEW

File: src/contact.html [hunk 1]
```diff
@@ -6,5 +6,13 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Document</title>
   </head>
-  <body></body>
+  <body>
+        <div class="header">
```
Comment (mz106) [inline src/contact.html:10]: Please change class name to header-contact

**Review by mz106 — CHANGES_REQUESTED** [review]
Please check comments

File: src/contact.html (hunk 1 above)
Comment (TobyLtt) [reply src/contact.html:10]: sure thing!

Comment (mz106) [reply src/contact.html:10]: thanks, good to go

**Review by mz106 — APPROVED** [review]
Good to merge

**Pull request merged**